src/tests/unit/platform/test_policy_ruleconfigs.py

```diff
@@ -601,3 +605,68 @@ def on_put(url, body, **kwargs):
     result_groups = rule_config.rule_groups
     assert len(result_groups) == 1
     assert result_groups[0].name == "Crapco_firewall"
+
+
+def test_copy_hbfw_rules(cbcsdk_mock):
+    """Tests the copy_rules_to function."""
+    put_called = False
+
+    def on_put(url, body, **kwargs):
+        nonlocal put_called
+        assert body == HBFW_COPY_RULES_PUT_REQUEST
+        put_called = True
+        return copy.deepcopy(HBFW_COPY_RULES_PUT_RESPONSE)
+
+    cbcsdk_mock.mock_request('PUT', '/policyservice/v1/orgs/test/policies/1492/rule_configs/host_based_firewall/_copy',
+                             on_put)
+    api = cbcsdk_mock.api
+    policy = Policy(api, 1492, copy.deepcopy(FULL_POLICY_5), False, True)
+    target_policy = Policy(api, 65536, copy.deepcopy(FULL_POLICY_1), False, True)
+    result = policy.host_based_firewall_rule_config.copy_rules_to(601, target_policy, "344")
```

Comment:
@avanbrunt-cb This is the test case that shows the various kinds of input to `copy_rules_to` (soon to be `copy_rules`).

Reply:
Understood. I'm more referring to our doc string and mentioning other.